### PR TITLE
Update component “notifiers”

### PR DESCRIPTION
**Pull request by itsmemin**
Automatically generated by Netlify CMS

### CODE REVIEW

File: src/_areas/design-system/components/notifiers.md [hunk 1]
```diff
@@ -1,230 +1,253 @@
...
 
-Used to give the user general information about the product or current process.
+Used to provide general information about the product or current process.
 
-### Success notifier
 
-![Success notifier](/images/components/notifier/success_notifier.svg)
 
-Used to inform the user that an action or workflow has been completed successfully.
+![Informative Notifier](/images/notifier-type-info.svg)
```
Comment (p-ob) [inline src/_areas/design-system/components/notifiers.md:59]:
```suggestion
![Info Notifier](/images/notifier-type-info.svg)
```
Thoughts on keeping this consistent with the type name?

File: src/_areas/design-system/components/notifiers.md [hunk 2]
```diff
@@ -1,230 +1,253 @@
...
 
 ### Animation
 
-Notifiers slide up and fade in as they appear on the screen, and slide down and fade out when they disappear.
+Notifiers slide up and fade in as they appear on the screen, and slide down and fade out as they disappear.
 
-![Notifier animation](/images/components/notifier/zui-notifier-animation-demo.gif)
-
----
+<center>
```
Comment (p-ob) [inline src/_areas/design-system/components/notifiers.md:113]: I don't think the `<center>` element here does anything. I do see it used in multiple markdown files, thought. Are we adding that, or is the CMS WYSIWYG editor doing it?

File: src/_areas/design-system/components/notifiers.md [hunk 3]
```diff
@@ -1,230 +1,253 @@
...
+## Responsive layout
 
-</GridCol>
+![Mobile notifier](/images/notifier-responsive.svg)
 
-<GridCol col="span-6">
+While the behavior remains unchanged, the location of Notifiers is different on mobile devices with viewports smaller than 720px. The Notifiers on mobile devices should be pinned to the bottom of the screen to make it easier to dismiss while holding the device.
 
-![Notifier information do not](/images/components/notifier/donot_indepth.svg)
+\[INSERT NOTE] The responsive behavior for Notifiers is not yet available in the toolkit.
```
Comment (p-ob) [inline src/_areas/design-system/components/notifiers.md:127]: If you want to wrap this in what the note HTML tag will be, we'll be able to magically enhance it when we ship the component and not need to revisit this content.

```suggestion
<docs-note>
    The responsive behavior for Notifiers is not yet available in the toolkit.
</docs-note>
```

File: src/_areas/design-system/components/notifiers.md [hunk 4]
```diff
@@ -1,230 +1,253 @@
...
 
-<DoNot />
+- - -
 
-A notifier should not be used to give the user in-depth information that requires more emphasis or to remain on the page, a well or dialog should be used in this case.
+## Best practices
 
-</GridCol>
+<Docs-grid columns="2">
```
Comment (p-ob) [inline src/_areas/design-system/components/notifiers.md:133]:
```suggestion
<docs-grid columns="2">
```

File: src/_areas/design-system/components/notifiers.md [hunk 5]
```diff
@@ -1,230 +1,253 @@
...
 
-<GridCol col="span-6">
+<docs-grid columns="2">
+<div>
 
-![Notifier action do not](/images/components/notifier/donot_actioncolor.svg)
+![Notifier action is the same color as the top bar](/images/do-notifier-color.svg)
 
-<DoNot />
+<docs-do>
```
Comment (p-ob) [inline src/_areas/design-system/components/notifiers.md:207]: Thoughts on changing these do/do nots into content related to "when to use actions"? Given just before, we describe when actions could be used, this seems like a great place to do that.

For example, I'm pretty sure I've seen a "confirmation notifier" (similar to our confirmation dialog pattern) for deleting that we'd never recommend.

File: src/_areas/design-system/components/notifiers.md [hunk 6]
```diff
@@ -1,230 +1,253 @@
...
-Multiple notifiers should be used sparingly and only seen in certain circumstances, for example:
+Multiple Notifiers should be used sparingly and only be seen in certain circumstances, for example:
 
-- multiple errors occurring on the same page.
-- multiple notifications appearing from the same workflow.
+* multiple errors occurring on the same page.
+* multiple notifications appearing from the same workflow.
 
----
+\[INSERT NOTE] The ability to have more than 1 Notifier on a screen is not yet in the toolkit.
```
Comment (p-ob) [inline src/_areas/design-system/components/notifiers.md:237]:
```suggestion
<docs-note>
The ability to have more than 1 Notifier on a screen is not yet in the toolkit. 
</docs-note>
```

File: src/_areas/design-system/components/notifiers.md (hunk 2 above)
Comment (itsmemin) [reply src/_areas/design-system/components/notifiers.md:113]: @p-ob I think the places we're using it is to center images that aren't 1000px on the page to match the rest. This image doesn't have a background, so it was shifted over to the left. We probably don't need an element for it if we can be sure that our GIFs have the additional padding to make them 1000px wide.

Comment (p-ob) [reply src/_areas/design-system/components/notifiers.md:113]: Ah, ok. So, TIL there _is_ a `<center>` element, but it's deprecated: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center.

I'm thinking we can create `<docs-center>` which simply wraps the contents with a tag with `text-align: center`; which ought to work? We can then do a find and replace to update docs, and going forward use the new one. I think enabling image width flexibility is a nice feature. 

@juliexiong @dave52 thoughts?

**Pull request merged**